R/calls_combine_by.R

```diff
@@ -0,0 +1,35 @@
+#' Combine calls by operator
+#'
+#' Combine list of calls by specific operator
+#'
+#' @param calls (`list` of calls)\cr
+#'   list containing calls to be combined by `operator`;
+#'   if empty, NULL is returned
+#' @param operator (`character(1)`)\cr
+#'   name/symbol of the operator passed as character string
+#'
+#' @return call or NULL, if `calls` is an empty list
+#'
+#' @examples
+#' \dontrun{
+#' calls <- list(
+#'   call_condition_choice("SEX", "F"),
+#'   call_condition_range("AGE", c(20, 50)),
+#'   call_condition_choice("ARM", "ARM: A"),
+#'   call_condition_logical("SURV", TRUE)
+#' )
+#' calls <- lapply(calls, str2lang)
```

Comment:
please don't use call_condition_* as they don't exist anymore. Add dummy calls using `quote` or something

Reply:
Documentation will be amended once all changes are settled on.